src/main/resources/lang/pylonbase/en.yml

```diff
@@ -259,6 +259,20 @@ item:
       <arrow> Makes crops, saplings, sugar cane, and cactus go brrr
       <arrow> <attr:Range:blocks>%range%</attr>
 
+  structure_locator_desert_pyramid:
+    name: "Structure Locator"
+    lore: |-
+      <arrow> Navigates to the nearest %struct%
+      <arrow> <insn>Right click</insn> to refresh
+      <arrow> <attr:Range:chunks>%range%</attr>
```

Comment:
`chunks` is not a registered `Quantity`, either add the quantity to core or use the `blocks` quantity